admin.go

```diff
@@ -167,6 +170,16 @@ func handleLoadConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleStop(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodPost {
```

Comment:
Maybe a little better would be

```go
if r.Method != http.MethodPost {
    http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
    return
}
```

Reply:
ok, do you want to handle anything other than post such as delete?

Reply:
Nah, POST is fine.